app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -20,8 +20,8 @@ def schedule_in_person_enrollment(user, pii)
         analytics(user: user).usps_ippaas_enrollment_created(
```

Comment:
I think we should update the constructor to accept `issuer` as well, so we can pass that to the `Analytics.new` call as `sp:`

Reply:
This doesn't get us what we want, though, because, `Analytics` doesn't automatically add a `service_provider` or `issuer` if you pass it a service provider object (it does add a `user_id` if you pass it a user).

Reply:
no sorry I meant we should pass `sp: service_provider.issuer`

Reply:
we resolved this face-to-face: the Analytics class doesn't currently support this case, but we'll add that support in an upcoming PR.

Reply:
for future us! specifically this part, where missing a `request` means `sp:` doesn't get merged in via `request_attributes`:

https://github.com/18F/identity-idp/blob/0cbae89050290123e50e6553960a70eb84f5741d/app/services/analytics.rb#L32